src/pages/resultsView/mutation/ResultsViewMutationMapperStore.ts

```diff
@@ -222,6 +226,31 @@ export default class ResultsViewMutationMapperStore extends MutationMapperStore
             }
         });
 
+        // Add numerical namespace column definitions.
```

Comment:
not necessary for this PR but this great example of something that should be in utility function and not bloating store